client/Pages/Home/Home.spec.tsx

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
`Pages` is meant for Page metadata - not components. Can this be moved to `Panels` (I know its only placeholder for now). 

Same is true for other files relating to `Home`